inc/languages/english/usercp.lang.php

```diff
@@ -291,7 +291,8 @@
 $l['date_uploaded'] ="Posted";
 $l['no_attachments_selected'] = "You did not select any attachments to delete.";
 $l['attachments_deleted'] = "The selected attachments have been successfully deleted.<br />You will be now redirected to the attachments manager.";
-$l['attachments_usage_quota'] = "- Using {1} of {2} ({3}) in {4} Attachments";
+$l['attachments_usage_percent'] = "({1}%)";
+$l['attachments_usage_quota'] = "- Using {1} of {2} {3} in {4} Attachments";
 $l['attachments_usage'] = "- {1} in {2} Attachments";
```

Comment:
No, it will create duplicate parenthesis.
As I stated earlier it has to be that way as the single variable is being used in 2 places ...

Reply:
```
./inc/languages/english/usercp.lang.php:$l['attachments_usage_quota'] = "- Using {1} of {2} ({3}) in {4} Attachments";
./usercp.php:		$usagenote = $lang->sprintf($lang->attachments_usage_quota, $friendlyusage, $attachquota, $percent, $totalattachments);
```

The language string is used only in one place, which is the one you have edited in this PR.

Reply:
@Shade- 
I don't know why I am failed you guys what the problem here.
If I keep it that way there will be ((20%)) << this kind of output as I have driven the second place a way that when displaying % is unnecessary it will not show (.....) instead thats why an additional lang var is created for the specific purpose.
'Unlimited' is not a % value. I felt evaluating a string for a variable meant to hold % value is just wrong.
You simulate the changes in your way and my way and will see what I say ...

Reply:
The thing is, `attachments_usage_quota` is not used with Unlimited; `attachments_usage` is used instead with the latter. Hence why `attachments_usage_percent` is useless, as it will be always filled with the percentage no matter if the user has unlimited attachment quota.

Reply:
Go to the end of the page: "Your Attachment Statistics"
Space used: 857.44 KB (2%)
Attachment Quota: 50 MB

Its good. But when set to unlimited:
Space used: 857.44 KB (**Unlimited**)
Attachment Quota: **Unlimited**

It is redundant, unnecessary to show quota in Space used value as next like shows the quota always dedicatedly. It is just there to evade the situation as there is no % lets repeat the quota ...

You get my point?
Now what I did, if there is Quota set it will show the % used as desired:
Space used: 857.44 KB (2%)
Attachment Quota: 50 MB

And when it is unlimited it will omit that
Space used: 857.44 KB 
Attachment Quota: **Unlimited**

I hope you get my point now. It is not about the lengthy state in thead. It is to handle this redundancy under "**Your Attachment Statistics** table.

Reply:
Alright, so `$percent` is used twice, not `$lang->attachments_usage_quota`. This is what I missed. Then, an extra blank space is displayed if the attachment quota is unlimited and this should be fixed.

Reply:
Thank you for getting my point finally.

Ya I know that I just ignored it being space is invisible.
There are changes I can't address extending to another PR. I don't know how to add the dot in the language file when there is no language variable. It has to be after the merge.

Reply:
A clean PR would be best if you can. I haven't tested the code yet and can't merge.

Reply:
Sure thing.

Reply:
@Shade- 
I recall why I ignored the extra space thing, coz when the attachment quota is unlimited `$l['attachments_usage']` is being used in place of `$l['attachments_usage_quota']`, displayed something like:
**Attachments Manager - 97.15 KB in 1 Attachments**
Thats all.
so there is no point addressing that space issue as the issue itself not being generated.

Reply:
If `$percent` is empty (when the quota is unlimited), an extra space is displayed [here](https://github.com/mybb/mybb/pull/3107/files#diff-05b404fc75b0e5bd6d57c79b667b44b8R12039).

Reply:
Yes. Of course.
Okay. will rectify.